packages/mongodb-mql-model/src/main/kotlin/com/mongodb/jbplugin/mql/BsonType.kt

```diff
@@ -109,10 +114,11 @@ fun Class<*>.toBsonType(): BsonType {
         Int::class.javaObjectType -> BsonAnyOf(BsonNull, BsonInt32)
         CharSequence::class.java, String::class.java -> BsonAnyOf(BsonNull, BsonString)
         Date::class.java, Instant::class.java, LocalDate::class.java, LocalDateTime::class.java ->
- BsonAnyOf(BsonNull, BsonDate)
+            BsonAnyOf(BsonNull, BsonDate)
```

Comment:
qq: is there something like prettier for kotlin projects. That probably should help with the formatting changes here.

Reply:
Yeah we have spotless in the Gradle project that does the formatting for us, but for some reason I need to check, it sometimes does weird things on the precommit hook.